src/touch.c

```diff
@@ -70,6 +71,11 @@ uint8_t touch_button_press(uint8_t touch_type)
     int16_t touch_sns;
     uint16_t exit_time_ms;
     uint16_t qt_led_toggle_ms;
+    uint16_t touch_thresh = 15; // hw version 1 / unknown
```

Comment:
I'd prefer to keep the contents in `touch.h`
`QTOUCH_TOUCH_THRESH_HW_V1_2 = 18;`
`QTOUCH_TOUCH_THRESH = 15;`

Reply:
Agreed. Done.